bin/inx-chronicle/src/collector/stardust_inx/mod.rs

```diff
@@ -264,42 +259,24 @@ impl HandleEvent<MilestoneState> for Solidifier {
                 ms_state.process_queue.pop_front();
             } else {
                 // Try the database
-                match self.db.get_block(block_id).await? {
-                    Some(block_rec) => {
-                        match block_rec
-                            .metadata
-                            .map(|metadata| metadata.referenced_by_milestone_index)
-                        {
-                            Some(ms_index) => {
-                                log::trace!("Block {} is referenced by milestone {}", block_id.to_hex(), ms_index);
+                match self.db.get_block_metadata(block_id).await? {
+                    Some(metadata) => {
+                        log::trace!(
+                            "Block {} is referenced by milestone {}",
```

Comment:
```suggestion
                            "Block `{}` is referenced by milestone `{}`",
```
I dunno....should we backtick identifiers consistently?

Reply:
I don't think we should be bothering with this tbh